sdk/identity/identity/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@azure/identity",
   "sdk-type": "client",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
What prompted the version bump of this package? We have never release the 1.0.1 version yet.

Reply:
We shipped 1.0.1 yesterday, depending on core-tracing preview6, which contained a bug that would surface at runtime when core-tracing preview5 was used by another package, even if the customer wasn't enabling tracing.

This was deemed too painful for our GA customers, so we unpublished this version which prevents us from publishing 1.0.1 again.